build-config/scripts/make-devices.pl

```diff
@@ -168,18 +168,11 @@ sub devices
     mkdir "${devdir}pts", 0755;
 
     print "NOR flash access: mtd mtdblock\n";
-    foreach my $i (0..7) {
+    foreach my $i (0..12) {
 	makedev("mtdblock${i}", "b", 31, ${i},     ${root}, ${root}, 0660);
 	makedev("mtd${i}",      "c", 90, (2*${i}), ${root}, ${root}, 0660);
     }
 
-    print "UBI device access: ubi\n";
-    makedev("ubi_ctrl", "c", 10, 63, ${root}, ${root}, 0660);
-    makedev("ubi0"    , "c", 253, 0, ${root}, ${root}, 0660);
-    foreach my $i (0..7) {
-	makedev("ubi0_${i}", "c", 253, ${i}+1, ${root}, ${root}, 0660);
-    }
-
```

Comment:
I think you are generally correct here also.

It would be cool if ONIE platforms that have NAND would do this on the fly for OS installers.  Somewhat discussed here:  http://lists.opencompute.org/pipermail/opencompute-onie/2015-April/000580.html

Something similar to how virtual block devices /dev/vd[a-z] are created:

https://github.com/opencomputeproject/onie/blob/master/rootconf/default/etc/init.d/makedev.sh#L93-112

You mentioned `mdev` -- maybe that is a better way to go.  I am not familiar with it.